site/utils/index.ts

```diff
@@ -238,8 +238,18 @@ export const removeDuplicates = <T, K extends keyof T>(arrayToRemoveDuplicates:
         (value, index, self) => index === self.findIndex((item) => item[key] === value[key]),
     );
 
-export const filterVehicleModes = (showUnderground?: boolean) =>
-    VEHICLE_MODES.filter((v) => (showUnderground ? true : v.value !== VehicleMode.underground));
+export const filterVehicleModes = (showUnderground?: boolean, consequenceType?: string) =>
+    VEHICLE_MODES.filter((v) => {
+        if (showUnderground) {
```

Comment:
Does this also need to check if `v === VehicleMode.underground`? Otherwise this filter will always return true if showUnderground is true and it won't reach the second if

Reply:
If I do this change it hides coach when show underground is true - so i don't think it is needed